<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderItems</class>
 <widget class="QWidget" name="OrderItems">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>20</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Items_L">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>0</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background: white;
	color: rgb(31, 48, 58);
	font: 600 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>item</string>
   </property>
  </widget>
  <widget class="QLabel" name="Price_L">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>0</y>
     <width>43</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background: white;
	color: rgb(31, 48, 58);
	font: 600 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Qty_L">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>0</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background: white;
	color: rgb(31, 48, 58);
	font: 600 10pt &quot;Segoe UI&quot;;
padding-left: 4px;</string>
   </property>
   <property name="text">
    <string>Qty</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
